tests/Knp/DoctrineBehaviors/ORM/ActivableTest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace Tests\Knp\DoctrineBehaviors\ORM;
+
+use Knp\DoctrineBehaviors\Reflection\ClassAnalyzer;
+use Doctrine\Common\EventManager;
+
+require_once 'EntityManagerProvider.php';
+
+class ActivableTest
+{
+    use EntityManagerProvider;
+
+    protected function getUsedEntityFixtures()
+    {
+        return array(
+            'BehaviorFixtures\\ORM\\ActivableEntity'
+        );
+    }
+
+    protected function getEventManager()
+    {
+        $em = new EventManager;
+
+        $em->addEventSubscriber(
+            new \Knp\DoctrineBehaviors\ORM\Activable\ActivableSubscriber(
+                new ClassAnalyzer(),
+                true,
+                'Knp\DoctrineBehaviors\Model\Activable\Activable'
+            ));
+
+        return $em;
+    }
+
+
+    /**
+     * @test
+     */
+    public function it_is_the_test_that_if_activation_is_setter()
```

Comment:
We use camelCase for PHPUnit test methods with testSomething naming convention here.
And this method name is not suitable here:
1. ```it_is_the_test_if``` ```is_too_looong``` and unnecessary.
2. You test a getter.
3. It is not PHPSpec.

```testEntityIsActiveByDefault()``` is more obvious here IMO.